docker-compose.yml

```diff
@@ -31,6 +31,7 @@ services:
       - BOPTEST_REDIS_HOST
       - BOPTEST_REGION
       - BOPTEST_TIMEOUT
```

Comment:
BOPTEST_TIMEOUT is really "idle" timeout. In other words how long there can be no interactions with a running test before it will automatically stop. It might be wise to rename and document these config variables a little better.